src/overlays/actors/ovl_En_Fg/z_en_fg.c

```diff
@@ -4,12 +4,38 @@
 
 #define THIS ((EnFg*)thisx)
 
+/*
+This may be a beta file of frogs, a version of them where they were all enemies
+*/
+
 void EnFg_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnFg_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnFg_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnFg_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void func_80A2D778(EnFg* this, GlobalContext* globalCtx);
+void EnFg_DoNothing(EnFg* this, GlobalContext* globalCtx);
+void func_80A2D9DC(EnFg* this, GlobalContext* globalCtx);
+void EnFg_AddDust(EnFgEffectDust* dustEffect, Vec3f* worldPos);
+void EnFg_UpdateDust(EnFgEffectDust* dustEffect);
+void EnFg_DrawDust(GlobalContext* globalCtx, EnFgEffectDust* dustEffect);
+
+extern u64 gDust1Tex[];
+extern u64 gDust2Tex[];
+extern u64 gDust3Tex[];
+extern u64 gDust4Tex[];
+extern u64 gDust5Tex[];
+extern u64 gDust6Tex[];
+extern u64 gDust7Tex[];
+extern u64 gDust8Tex[];
```

Comment:
Could you leave the addresses of those textures somewhere?
Having the addresses is useful to decompile the object and and properly identify what is what.

Reply:
+1 on this

Reply:
Is it enough that the addresses are already in object_script.txt? If not, happy to comment the addresses

Reply:
From my previous experience doing object, it is much better having the address in the same actor for doing this. So, yeah, please add a comment with the addresses.